kibana/values.yaml

```diff
@@ -120,10 +129,20 @@ readinessProbe:
   successThreshold: 3
   timeoutSeconds: 5
 
-imagePullSecrets: []
+
 nodeSelector: {}
 tolerations: []
 affinity: {}
 
-nameOverride: ""
-fullnameOverride: ""
+# To export a dashboard from a running Kibana 6.3.x use:
+# curl --user <username>:<password> -XGET https://kibana.yourdomain.com:5601/api/kibana/dashboards/export?dashboard=<some-dashboard-uuid> > my-dashboard.json
+# A dashboard is defined by a name and a string with the json payload or the download url
+dashboardImport:
+  enabled: false
+  timeout: 60
+  xpackauth:
+    enabled: false
+    username: myuser
```

Comment:
Its not a good idea to store secrets directly in the values.yaml. If you take a look at the readinessProbe for the Kibana container there is a function that can be used to properly use http/https and the right authentication without needing to duplicate the password and add it to the values file.

https://github.com/elastic/helm-charts/blob/a3865369ab6a4a257453145a6d20b1d8266f6a94/kibana/templates/deployment.yaml#L94-L106